source/docs/getting-started/kitchen-yml.html.md

```diff
@@ -0,0 +1,147 @@
+---
+title: ".kitchen.yml"
+---
+
+##### .kitchen.yml
+
+Let's turn our attention to the `.kitchen.yml` file for a minute. While ChefDK may have created the initial file automatically, it's expected that you will read and edit this file. After all, you know what you want to test... right? Opening this file in your editor of choice we see something like the following:
+
+~~~yaml
+---
+driver:
+  name: vagrant
+
+provisioner:
+  name: chef_zero
+  # You may wish to disable always updating cookbooks in CI or other testing environments.
+  # For example:
+  #   always_update_cookbooks: <%= !ENV['CI'] %>
+  always_update_cookbooks: true
+
+verifier:
+  name: inspec
+
+platforms:
+  - name: ubuntu-16.04
+  - name: centos-7.3
+
+suites:
+  - name: default
+    run_list:
+      - recipe[git_cookbook::default]
+    verifier:
+      inspec_tests:
+        - test/smoke/default
+    attributes:
+~~~
+
+Very briefly we can cover the common sections you're likely to find in a `.kitchen.yml` file:
+
+* `driver`: Set and configure the driver. We're explicitly setting it via `name: vagrant` even though this is the default.
+* `transport`: Configure settings for the transport layer (SSH/WinRM). No configuration is needed of this for our guide as default values suffice.
+* `provisioner`: This tells Kitchen how to run Chef, to apply the code in our cookbook to the machine under test.  The default and simplest approach is to use `chef-solo`, but other options are available, and ultimately Test Kitchen doesn't care how the infrastructure is built - it could theoretically be with Puppet, Ansible, or Perl for all it cares.
+* `verifier`: This is where we configure the behaviour of the Kitchen Verifier - this component that is responsible for executing tests against the machine after Chef has converged.
+* `platforms`: This is a list of operation systems on which we want to run our code. Note that the operating system's version, architecture, cloud environment, etc. might be relevant to what Test Kitchen considers a **Platform**.
+* `suites`: This section defines what we want to test.  It includes the Chef run-list and any node attribute setups that we want run on each **Platform** above. For example, we might want to test the MySQL client cookbook code separately from the server cookbook code for maximum isolation.
+
+Let's say for argument's sake that we only care about running our Chef cookbook on Ubuntu 16.04 with Chef version 13. In that case, we can edit the `.kitchen.yml` file so that we add `require_chef_omnibus: 13` under the provisioner and trim the list of `platforms` to only one entry like so:
```

Comment:
This reinstalls chef on top of itself everytime at present. I'm going to leave it out of the next commit but we can still change this if my testing goes well.